docs/QuickStart.rst

```diff
@@ -115,7 +115,7 @@ Example to build ICON for CPU with NVHPC:
 
 .. code-block:: console
 
-    $ SPACK_TAG=$(cat "config/cscs/SPACK_TAG")
+    $ SPACK_TAG=$(cat "config/cscs/SPACK_TAG_DAINT")
```

Comment:
This is not correct for Balfrin.

Reply:
Well it's an "Example to build ICON for CPU with NVHPC". Higher up it's written as "SPACK_TAG_*". I thought that makes it clear.

Reply:
I changed the line above the comment now to "Example to build ICON for CPU with NVHPC on Piz Daint". Is it clearer like this?

Reply:
Yes. I guess we can assume users can extrapolate to other machines.